cfe_internal/update/update_policy.cf

```diff
@@ -95,7 +95,7 @@ bundle agent cfe_internal_update_policy
       "$(inputs_dir)/cf_promises_validated"
       comment => "Check whether a validation stamp is available for a new policy update to reduce the distributed load",
       handle => "cfe_internal_update_policy_check_valid_update",
-      copy_from => u_rcp("$(master_location)/cf_promises_validated", @(update_def.policy_servers)),
+      copy_from => u_rcp("$(master_location)/cf_promises_validated", @(update_def.policy_servers), "$(sys.policy_hub_port)"),
```

Comment:
Assumes all servers in the list will use the same port. Might it be better if servers element could take the form of 10.0.0.0 or 10.0.0.0:5309 and do away with the port parameter.